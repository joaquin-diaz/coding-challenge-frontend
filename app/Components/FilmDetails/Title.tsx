import styled from '@emotion/styled';
import styles from '../../constants/styles';

export default styled.h1`
  background-color: ${styles.colors.primary};
  color: white;
  margin: 0;
  padding: 20px;
  text-align: center;
  font-weight: 500;
`;
