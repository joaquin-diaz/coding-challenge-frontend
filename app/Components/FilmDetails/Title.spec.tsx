import * as React from 'react';

import Title from './Title';
import { mount } from 'enzyme';

describe('<Title />', () => {
  it('should render', () => {
    const wrapper = mount(<Title>Test</Title>);
    expect(wrapper).toMatchSnapshot();
  });
});
