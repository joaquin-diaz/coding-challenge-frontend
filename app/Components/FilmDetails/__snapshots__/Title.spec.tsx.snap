// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render 1`] = `
.emotion-0 {
  background-color: #3f51b5;
  color: white;
  margin: 0;
  padding: 20px;
  text-align: center;
  font-weight: 500;
}

<Styled(h1)>
  <h1
    className="emotion-0"
  >
    Test
  </h1>
</Styled(h1)>
`;
